archimedes/registry.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2014-2018 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#
+# Authors:
+#   Valerio Cosentino <valcos@bitergia.com>
+#
+
+import json
+import logging
+import os
+
+from archimedes.errors import NotFoundError, RegistryError
+from archimedes.meta_obj import MetaObj
+from archimedes.utils import load_json
+
+logger = logging.getLogger(__name__)
+
+REGISTRY_NAME = ".registry"
+
+
+class Registry:
+
+    def __init__(self, root_path):
+        """This class allows to manipulate the Kibana objects using aliases, thus avoiding the user
```

Comment:
Please improve the documentation following the rules defined in https://www.python.org/dev/peps/pep-0257/#id17

Reply:
it seems that this comment applies to all archimedes classes, so I'll address this in another PR

Reply:
I'll revamp the PR https://github.com/Bitergia/archimedes/pull/7 to improve the doc.